declaration.tsconfig.json

```diff
@@ -0,0 +1,9 @@
+
+{
+  "extends": "./tsconfig",
+  "compilerOptions": {
+    "declaration": true,
+    "noEmit": false,
+    "emitDeclarationOnly": true
+  }
+}
```

Comment:
```suggestion
{
  "extends": "./tsconfig",
  "compilerOptions": {
    "declaration": true,
    "noEmit": false,
    "emitDeclarationOnly": true
  }
}
```

Reply:
Thanks 🙏 I wonder why my EditorConfig VSCode extension didn't catch / fix this 🤔